languages/python/exercises/concept/chaitanas-colossal-coaster/.docs/introduction.md

```diff
@@ -31,231 +31,144 @@ Another way to add an item to the list is to `.insert()` an item. This method gi
-#=> ["Bruce", "Natasha", "Thor", "Tony"]
+>>> names = ["Tony", "Natasha", "Thor", "Bruce"]
+>>> names.sort()
+>>> names
+["Bruce", "Natasha", "Thor", "Tony"]
 ```
 
-If you want the sort to be in descending order, you can use the reverse parameter.
+If you want the sort to be in descending order, you can use the reverse parameter:
 
 ```python
-ex_list = ["Tony", "Natasha", "Thor", "Bruce"]
-ex_list.sort(reverse=True)
-print(ex_list)
-#=> ["Tony", "Thor", "Natasha", "Bruce"]
+>>> names = ["Tony", "Natasha", "Thor", "Bruce"]
+>>> names.sort(reverse=True)
+>>> names
+["Tony", "Thor", "Natasha", "Bruce"]
 ```
 
 For cases where changing your original list is undesirable, the built-in [`sorted()`][sorted] can be used to return a new, sorted copy of your original list.
 
-### Occurrences of an item in the list.
+### Occurrences of an item in the list
 
 You can find the number of occurrences of an element in the list with the help of `.count()`. It takes the element you need to tally as its argument, and returns the total number of times it appears on the list.
 
 ```python
-ex_list = [1, 4, 7, 8, 2, 9, 2, 1, 1, 0, 4, 3]
-ex_list.count(1)
-#=> 3
+>>> items = [1, 4, 7, 8, 2, 9, 2, 1, 1, 0, 4, 3]
+>>> items.count(1)
+3
 ```
 
-### Finding the index of items.
-
-`.index()` will provide you the index number of the first occurrence of the item you pass in. If you do not have any occurrences of the item, a `ValueError` is raised. If you do not need the exact position of an item and are only checking that it is present on the list, the built-in `in` operator is more efficient.
+### Finding the index of items
 
-Index starts with 0.
+The function `.index()` will provide you the index number of the first occurrence of the item you pass in. If you do not have any occurrences of the item, a `ValueError` is raised. If you do not need the exact position of an item and are only checking that it is present on the list, the built-in `in` operator is more efficient. The index is zero-based, meaning the first item is at the position 0.
 
 ```python
-ex_list = [1, 4, 7, 8, 2, 9, 2, 1, 1, 0, 4, 3]
-ex_list.index(4)
-#=> 1
-ex_list.index(10)
-#=>ValueError: 10 is not in list
+>>> items = [7, 4, 1, 0, 2, 5]
+>>> items.index(4)
+1
+>>> items.index(10)
+ValueError: 10 is not in list
 ```
 
 You can provide start and end indices to search within a specific section of the list.
 
 ```python
-ex_list = [1, 4, 7, 8, 2, 9, 2, 1, 1, 0, 4, 3]
-ex_list.index(4, 2, 12)
-#=> 10
+>>> names = ["Tina", "Leo", "Thomas", "Tina", "Emily", "Justin"]
+>>> names.index("Tina")
+0
+>>> names.index("Tina", 2, 5)
+3
 ```
 
 ### Making Copies
 
-Remember that _names_ in python are just labels that reference an underlying object. This creates a few surprises when working with lists.
+Lists are collections of items which are referenced by an index. Therefor, if you do re-assign a list object to another variable, any change you do to the new variable will also impact the original variable.
```

Comment:
```suggestion
Remember that _names_ (variables) in Python are labels that point to underlying objects. This creates a few surprises when working with lists.  

If you assign a list object to a new variable name, any change you make to the list using the _new_ variable name will _also impact_ the original variable name -- both names point to the **same list object**.  To avoid this problem, use `copy()` or a slice.
```